### PR TITLE
Fix #2282: Added RTL Support to topic lessons tab

### LINKED ISSUE #2282: [RTL] High-fi Topic Lessons Tab
`Topic Lessons` needs RTL support as shown in below screen.
TP (Lessons, Auto-Scroll Top) – RTL

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by priyanka0906**
## Explanation
fixes #2282 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

## Before:
Mobile Portrait | Mobile Landscape
------------ | -------------
Screenshot_1611496064|Screenshot_1611496077

## After:
Mobile Portrait | Mobile Landscape
------------ | -------------
Screenshot_1611394972|Screenshot_1611413644

### CODE REVIEW

File: app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt [hunk 1]
```diff
@@ -8,7 +8,10 @@ import android.util.AttributeSet
 import android.util.TypedValue
 import android.view.View
 import androidx.core.content.ContextCompat
+import androidx.core.text.TextUtilsCompat
+import androidx.core.view.ViewCompat
 import org.oppia.android.R
+import java.util.*
```
Comment (FareesHussain) [inline app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt]: Wildcard import ktlint error
```suggestion
import java.util.Locale
```

File: app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt [hunk 2]
```diff
@@ -74,6 +79,8 @@ class SegmentedCircularProgressView : View {
   }
 
   override fun onDraw(canvas: Canvas) {
+    if(isRTL)
+      rotationY=180f
```
Comment (FareesHussain) [inline app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt]:
```suggestion
    if (isRTL)
      rotationY = 180f
```

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@priyanka0906 Added few suggestions also add screenshots before this implementation for better review

**Comment by rt4914** [comment]
Apart from what @FareesHussain has suggestion update your title to
`Fix #2282: Added RTL Support to topic lessons tab`

**Comment by priyanka0906** [comment]
> @priyanka0906 Added few suggestions also add screenshots before this implementation for better review

screenshots of the code??

**Comment by FareesHussain** [comment]
> > @priyanka0906 Added few suggestions also add screenshots before this implementation for better review
> 
> screenshots of the code??

No. Screenshot of the layout without your implementation and screenshot of the layout with your implementation to compare the changes

**Comment by priyanka0906** [comment]
> > > @priyanka0906 Added few suggestions also add screenshots before this implementation for better review
> > 
> > 
> > screenshots of the code??
> 
> No. Screenshot of the layout without your implementation and screenshot of the layout with your implementation to compare the changes

okay

**Comment by priyanka0906** [comment]
> > > > @priyanka0906 Added few suggestions also add screenshots before this implementation for better review
> > > 
> > > 
> > > screenshots of the code??
> > 
> > 
> > No. Screenshot of the layout without your implementation and screenshot of the layout with your implementation to compare the changes
> 
> okay

is it fine now?

**Comment by priyanka0906** [comment]
> Apart from what @FareesHussain has suggestion update your title to
> `Fix #2282: Added RTL Support to topic lessons tab`

done

File: app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt (hunk 2 above)
Comment (priyanka0906) [reply app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt]: done

File: app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt (hunk 1 above)
Comment (priyanka0906) [reply app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt]: @FareesHussain  done

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**